src/main/java/org/broadinstitute/hellbender/tools/exome/TargetAnnotation.java

```diff
@@ -11,14 +11,21 @@
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
  */
 public enum TargetAnnotation {
-
     /**
      * Fraction of bases that are G or C in the reference overlapping the target.
      *
      * <p>This value is always a number between 0 and 1 except in very rare cases with small targets with all, undefined bases (N or X),
-     * in this case it value will be {@link Double#NaN NaN}</p>.
+     * in this case it value will be {@link Double#NaN NaN}.</p>
+     */
+    GC_CONTENT(TargetTableColumn.GC_CONTENT),
+
+    /**
+     * Number of baits overlapping with the target. Only relevant in the context of target-enriched (e.g. WES) sequencing data.
+     *
+     * <p>This value is never {@link Double#NaN} and is non-negative. It can be zero or fractional (if a bait is shared by more
+     * than a single target).</p>
```

Comment:
Not many, about 1% at most. They come in contiguous clusters where targets are "hypersegmented". For example:
```
9	133710699	133710732	CEX-chr9-133710699-133710732	0.8333333333333333
9	133710743	133710782	CEX-chr9-133710743-133710782	1.1666666666666665
9	133710787	133710817	CEX-chr9-133710787-133710817	0.6666666666666666
9	133710818	133710914	CEX-chr9-133710818-133710914	1.3333333333333333
```

There are also caveats to uniform bait-sharing too. For example, there are numerous cases like:
```
19	54724953	54725253	CEX-chr19-54724953-54725253	4.0
19	54725658	54725658	CEX-chr19-54725658-54725658	0.5
19	54725698	54726004	CEX-chr19-54725698-54726004	5.5
19	54726148	54726436	CEX-chr19-54726148-54726436	4.0
```
Here, the bait responsible for pulling `CEX-chr19-54725658-54725658` probably has a small overlap with `CEX-chr19-54725698-54726004` and as a result, the latter has absorbed half of its weight.

A reasonable bait-based coverage collection solves this issue. This PR is a temporary fix.